models/games.js

```diff
@@ -60,5 +60,17 @@ exports.insertCommentByReviewId = (newComment) => {
     })
 };
 
+exports.updateReview = (updateVotes, review_id) => {
+    return this.selectReviewById(review_id).then(() => {
```

Comment:
This check doesnt need to be here, If you give this query an invalid review_id you can handle that error after based on the response.

Reply:
unsure on what you mean by this. This is just me reusing a function to test if the review_id exists, before continuing, as Liam was doing in the lecture yesterday

Reply:
What was showed in the lecture is correct, but for this endpoint using a helper function is unnecessary. Simply checking the result and throwing an error based off of that would be more than enough.

Reply:
Hi, when trying to remove this check, there is no error as the sql will just update nothing. Are you saying to also remove this test?